extensions/amp-analytics/0.1/test/test-analytics-root.js

```diff
@@ -373,6 +373,16 @@ describes.realWin('AmpdocAnalyticsRoot', {amp: 1}, (env) => {
         expect(children).to.deep.equal([child, child2]);
       });
 
+      it('should not find elements with data-vars-* if useDataVars is false', async () => {
+        const getElementSpy = env.sandbox.spy(root, 'getDataVarsElements_');
+        child.classList.add('myClass');
+        child2.classList.add('myClass');
+        expect(
+          await root.getElements(body, ['.myClass'], null, false)
+        ).to.deep.equal([child, child2]);
+        expect(getElementSpy).to.be.callCount(0);
+      });
+
```

Comment:
Can we also apply this test to the `EmbedAnalyticsRoot` tests (line 549)

Reply:
Done!